openpype/hosts/maya/plugins/load/load_arnold_standin.py

```diff
@@ -6,23 +6,29 @@
 from openpype.settings import get_project_settings
 from openpype.pipeline import (
     load,
+    legacy_io,
     get_representation_path
 )
 from openpype.hosts.maya.api.lib import (
-    unique_namespace, get_attribute_input, maintained_selection
+    unique_namespace,
+    get_attribute_input,
+    maintained_selection,
+    convert_to_maya_fps
 )
 from openpype.hosts.maya.api.pipeline import containerise
 
-
 def is_sequence(files):
     sequence = False
-    collections, remainder = clique.assemble(files)
+    collections, remainder = clique.assemble(files, minimum_items=1)
     if collections:
         sequence = True
-
     return sequence
 
 
+def get_current_session_fps():
+    session_fps = float(legacy_io.Session.get('AVALON_FPS', 25))
```

Comment:
@iLLiCiTiT is this `legacy_io` usage ok, or is there already an alternative to this?